File: Python Word Examples/ParagraphsApiDemo/addParagraph.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.ParagraphsApi(configuration)

file_name = "AddParagraph.docx"
node_path = "Section/0/Body/0"
extra = {'folder': 'input','index_of_paragraph': 0,'text': 'E-iceblue','dest_file_path': 'output/AddParagraph_output.docx'}
api.add_paragraph(file_name, node_path, **extra)

File: JavaScript Word Examples/ParagraphsApiDemo/addParagraph.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.ParagraphsApi(apiClient);

    var filename = "AddParagraph.docx";
    var nodePath = "Section/0/Body/0";
    var opts = {
        folder: 'input',
        indexOfParagraph: 0,
        text: "E-iceblue"
    };
    var destFilePath = "output/AddParagraph_output.docx";
    instance.addParagraph(filename, nodePath, destFilePath, opts, function (error, data, response) {
        if (error) throw error;
        });
})();